frontend/src/v5/ui/components/shared/modalsDispatcher/templates/infoModal/authenticatingModal/authenticatingModal.component.tsx

```diff
@@ -14,18 +14,18 @@
  *  You should have received a copy of the GNU Affero General Public License
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import { useEffect, useRef } from 'react';
 import { InfoModal } from '../../infoModal/infoModal.component';
 import { formatMessage } from '@/v5/services/intl';
+import { useEffect, useRef } from 'react';
 import { useLocation } from 'react-router';
 
 export const AuthenticatingModal = ({ onClickClose }) => {
-	const location = useLocation();
-	const initialLocation = useRef(location);
+	const currentLocation = useLocation();
+	const initialLocation = useRef(currentLocation);
 
 	useEffect(() => {
-		if (location !== initialLocation.current) onClickClose();
-	}, [location]);
+		if (currentLocation.pathname !== initialLocation.current.pathname) onClickClose();
+	}, [currentLocation]);
```

Comment:
Nice. This doesn't have to be done in this PR but I think this should be moved to the base modal component so it does this for _all_ modals.

I made an issue for this a while ago https://github.com/3drepo/3D-Repo-Product-Team/issues/441